packages/cli/cli/src/loaders/fractal.test.js

```diff
@@ -0,0 +1,25 @@
+const fractal = require('@frctl/fractal');
+const {expect} = require('../../../../../test/helpers');
+const loader = require('./fractal');
+
+describe('CLI fractal loader', function () {
+  it('exports a function', function () {
+    expect(loader).to.be.a('function');
+  });
+
+  it('returns a Fractal instance', function () {
+    expect(loader()).to.be.an.instanceOf(fractal.Fractal);
+  });
+
+  // it('attempts to load the fractal version from the CWD first', function () {
```

Comment:
Actually I was unsure how to go about writing a test this... any ideas on the best way to do it?